src/app/routes/article/utils/getOnwardsData.ts

```diff
@@ -81,7 +81,7 @@ const getOnwardsPageData = async ({
   variant?: string;
   isAdvertising: boolean;
   isArticleSfv?: boolean;
-  agent: Agent | null;
+  agent: void | Agent | null;
```

Comment:
What does `void` mean here? I imagine this type previous meant: 'an agent or null'?

Reply:
Going to test removing this.

Reply:
Removed without issue.